.github/workflows/ci_cd.yml

```diff
@@ -33,7 +33,6 @@ jobs:
         uses: ansys/actions/doc-style@v5
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          vale-config: "doc/.vale.ini"
           vale-version: "3.3.0"
```

Comment:
@dipinknair I see we are using Vale version 3.3.0. Once we figure out what's going on, we should update this dev guide topic: https://dev.docs.pyansys.com/content-writing/content-how-tos/work-around-Vale.html#specify-a-specific-vale-version